sdk/search/azure-search/src/test/java/com/azure/search/SearchIndexClientBuilderTests.java

```diff
@@ -131,123 +130,41 @@ public void whenBuildClientAndVerifyPropertiesThenSuccess() {
     }
 
     @Test
-    public void verifyEmptyEndpointIsInvalidAsyncTest() {
-        expectThrowsWithMessage("'endpoint' must be a valid URL", () -> new SearchIndexClientBuilder()
-            .endpoint("")
-            .credential(searchApiKeyCredential)
-            .searchServiceVersion(apiVersion)
-            .indexName(indexName)
-            .buildAsyncClient());
+    public void emptyEndpointThrowsIllegalArgumentException() {
```

Comment:
To my point of view, error message change is also a breaking change. To verify the message right here, make sure developer pay attention when they make a change to the error message.
I prefer to assert the message as well.

Reply:
Exception messages are treated on a per case basis on whether they are considered a breaking change, with the general decision being would this message be programmatically consumed. Given that this is a runtime exception that isn't networking related I believe this would be unlikely so there isn't a strong conviction on testing the message of the exception.